src/main/java/org/generationcp/middleware/manager/ontology/OntologyVariableDataManagerImpl.java

```diff
@@ -984,6 +1014,11 @@ public boolean areVariablesUsedInStudy(final List<Integer> variablesIds) {
 		return !query.list().isEmpty();
 	}
 
+	@Override
+	public boolean areVariablesUsedInAttributes(final List<Integer> variablesIds) {
```

Comment:
This is duplicate of `isVariableUsedInGermplasm`?

Reply:
done.